lib/cfa/augeas_parser.rb

```diff
@@ -89,12 +131,20 @@ def []=(key, value)
       end
     end
 
+    # @param matcher [Matcher]
+    # @return [Array<AugeasElement>] matching elements
+    # @todo for consistency this should allow &matcher too
     def select(matcher)
       @data.select(&matcher)
     end
 
     # @note for internal usage only
     # @private
+    #
+    # Initializes {#data} from *prefix* in *aug*.
+    # @param aug [::Augeas]
+    # @param prefix [String] Augeas path prefix
+    # @return [void]
```

Comment:
We have disagreed on this before, I know. You are not documenting because users cannot use it anyway, I am documenting for the maintainers.